tools/build/workspace.bzl

```diff
@@ -158,6 +158,11 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
         commit = "e562960fe303c0ffab6f3458fcdb1544b56fd81e",
         build_file = "@gapid//tools/build/third_party:llvm.BUILD",
         sha256 = "3ef3d905849d547b6481b16d8e7b473a84efafbe90131e7bc90a0c6aae4cd8e6",
+	# This patch fixes missing standard library includes which leads to compilation
+	# issues in recent gcc. This issue is fixed on recent llvm versions (since
+	# https://github.com/llvm-mirror/llvm/commit/e0402b5c9813a2458b8dd3f640883110db280395),
+	# but updating our llvm version leads to other errors.
+	patch_file = "@gapid//tools/build/third_party:llvm_fix.patch",
```

Comment:
fix the indent.

Personally, this is a bit of a borderline. We don't really need to update to gcc10, and you are adding a maintenance burden here with the patch. It's a small patch, so I can live with it.

Reply:
I agree, I would also prefer to update LLVM than to use a patch. But we would have to go from llvm-mirror to llvm-project, and that effort would not be justified to support gcc-10.

The indent looked right in the editor, because I copied that part from the grpc entry, which uses tabs. I replaced all tabs in the file with spaces.